plugin.go

```diff
@@ -53,6 +53,7 @@ type Step struct {
 	Env       map[string]string `yaml:"env,omitempty"`
 	Async     bool              `yaml:"async,omitempty"`
 	SoftFail  interface{}       `json:"soft_fail" yaml:"soft_fail,omitempty"`
+  Notify    interface{}       `yaml:"notify,omitempty"`
```

Comment:
Is it possible to add proper type with possible values? Will serve as a documentation and also validation.